buffalo/cmd/dev.go

```diff
@@ -92,7 +92,13 @@ func startWebpack(ctx context.Context) error {
 		}
 	}
 
-	cmd := exec.CommandContext(ctx, webpack.BinPath, "--watch")
+	if _, err := os.Stat(webpack.DevServerPath); err != nil {
+		if os.IsNotExist(err) {
```

Comment:
for me the error that was returned was:

```go
### err -> &os.PathError{
    Op:   "stat",
    Path: "node_modules/.bin/webpack-dev-server",
    Err:  syscall.Errno(0x2),
}
```

I think it's safe enough to say any error returned from `os.Stat` should be good enough.